install/ossm-cert-manager.adoc

```diff
@@ -10,37 +10,27 @@ toc::[]
 
 The cert-manager tool is a solution for X.509 certificate management on Kubernetes. It delivers a unified API to integrate applications with private or public key infrastructure (PKI), such as Vault, Google Cloud Certificate Authority Service, Let's Encrypt, and other providers.
 
+The cert-manager tool ensures that the certificates are valid and up-to-date by attempting to renew certificates at a configured time before they expire.
+
 [IMPORTANT]
 ====
 The cert-manager tool must be installed before you create and install your `Istio` resource.
 ====
 
-The cert-manager tool ensures the certificates are valid and up-to-date by attempting to renew certificates at a configured time before they expire.
-
 include::modules/ossm-about-cert-manager.adoc[leveloffset=+1]
-include::modules/ossm-installing-cert-manager.adoc[leveloffset=+1]
-
-.Next steps
-To install `istio-csr`, you must follow the `istio-csr` installation instructions for the type of update strategy you want. By default, `spec.updateStrategy` is set to `InPlace` when you create and install your `Istio` resource. You create and install your `Istio` resource after you install `istio-csr`.
-
-* xref:../install/ossm-cert-manager.adoc#inplace-istio-csr-installation_ossm-cert-manager[Installing the istio-csr agent by using the in place update strategy]
-* xref:../install/ossm-cert-manager.adoc#revision-based-istio-csr-installation_ossm-cert-manager[Installing the istio-csr agent by using the revision based update strategy]
-
-include::modules/ossm-cert-manager-istio-csr-inplace-update-strategy.adoc[leveloffset=+2]
+include::modules/ossm-installing-cert-manager.adoc[leveloffset=+2]
```

Comment:
We already know that a single line space is required for DITA, so make a note to fix these later (and use a line space between in future content). Not a merge blocker, just FYI.